indigenous-acknowledgement.md

```diff
@@ -0,0 +1,39 @@
+# Linux Australia's Indigenous Acknowledgement
+
+## Overview
+
+Linux Australia recognises Australia's Indigenous heritage, and the history of inequality and under-representation faced by Indigenous Australians in all sectors of society, including the open source community. Linux Australia, in alignment with its core values [0], seeks to ensure the recognition and representation of Indigenous people and culture throughout its activities, events and administration mechanisms.
+
+This policy instrument specifies the touchpoints at which recognition and representation of Indigenous persons and culture are to be built into the organisation.
+
+This policy instrument recognises the disctinction between:
+
+* Welcome to Country: A ceremony of significant cultural importance which may only be undertaken by an Elder of an Indigenous community, and;
+* Acknowledgement of Country: An acknowledgement of the traditional ownership of a site, respecting Indigenous Elders past, present and future.
+
+## Administrative activities
+
+* Linux Australia's Annual General Meeting (AGM) will include an Acknowledgement of Country, provided by an Office Bearer of Linux Australia. This shall be formally minuted.
+
+* Linux Australia's website will contain an appropriate Indigenous Acknowlegement, updated from time to time to reflect appropriate cultural protocol.
+
+## Events run under the auspices of Linux Australia
+
+* Each event run under the auspices of Linux Australia, will at a minimum, provide an Acknowlegement of Country in the opening of the event. The Acknowledgement of Country will be made by an event organiser, event lead, or person of authority in line with the protocols of the local Indigenous community.
+
+* At the discretion of the Event's Subcommittee, the Event may choose to invite an Indigenous Elder to provide a Welcome to Country for the event, in line with the protocols observed by the Indigneous community of the event's geographical location.
```

Comment:
Not all events would have subcommittees.

"At the discretion of the Event's Organisers" - assumes that all events have organisers but this could fail to make a distinction between those on the ground running about versus, well, a subcommittee/committee.